net/nuster/Makefile

```diff
@@ -0,0 +1,177 @@
+#
+# Copyright (C) 2010-2016 OpenWrt.org
+# Copyright (C) 2009-2016 Thomas Heil <heil@terminal-consulting.de>
+# Copyright (C) 2018 Christian Lachner <gladiac@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=nuster
+PKG_VERSION:=1.7.10.1
+PKG_RELEASE:=01
+
+PKG_SOURCE:=nuster-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/jiangwenyuan/nuster/releases/download/v$(PKG_VERSION)/
+PKG_HASH:=0cf75205ef39db10700abc4307cab5fa410d612c8b096d14adb99cd8775b22d1
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(BUILD_VARIANT)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_LICENSE:=GPL-2.0
+MAINTAINER:=Christian Lachner <gladiac@gmail.com>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/nuster/Default
+  SUBMENU:=Web Servers/Proxies
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=A caching proxy server based on HAProxy
+  URL:=https://github.com/jiangwenyuan/nuster/
+endef
+
+define Download/lua534
+	FILE:=lua-5.3.4.tar.gz
+	URL:=https://www.lua.org/ftp/
+	HASH:=f681aa518233bc407e23acf0f5887c884f17436f000d453b2491a9f11a52400c
+endef
+
+define Build/Prepare
+	$(call Build/Prepare/Default)
+ifeq ($(ENABLE_LUA),y)
+	tar -zxvf $(DL_DIR)/lua-5.3.4.tar.gz -C $(PKG_BUILD_DIR)
+	ln -s $(PKG_BUILD_DIR)/lua-5.3.4 $(PKG_BUILD_DIR)/lua
```

Comment:
wat? You're going to itnernally download, build and install your own lua 5.3 on the side?

Reply:
Hey there,
yes, this is what we also do for the haproxy-package (nuster is pretty much just haproxy with added cache support) since we rely on a rather current lua library.
Aaaanyway, this PR is rather old and my initial nuster package clearly needs more work since nuster had many new releases and the Makefile still has the obsolete @OPENSSL_ dependencies.
Closing this one.